### DISCUSSION TITLE
可否添加暗色模式

### DISCUSSION THREAD

**Discussion by AlphaLuckyyy** [leiurayer/downkyi#1025] [Ideas]
感谢作者和其他维护人员!

**Comment by yaobiao131** [comment]
在我的跨平台版本计划中





------------------ 原始邮件 ------------------
发件人: AlphaLuckyyy ***@***.***&gt;
发送时间: 2024年1月30日 01:05
收件人: leiurayer/downkyi ***@***.***&gt;
抄送: Subscribed ***@***.***&gt;
主题: Re: [leiurayer/downkyi] 可否添加暗色模式 (Discussion #1025)
